src/platforms/react-native/manual-setup/sourcemaps.mdx

```diff
@@ -32,8 +36,9 @@ node_modules/@sentry/cli/bin/sentry-cli releases \
     files RELEASE_NAME \
     upload-sourcemaps \
     --dist DISTRIBUTION_NAME \
-    --strip-prefix /path/to/project/root \
-    /path/to/your/files
+    --url-prefix app:/// \
+    --bundle index.android.bundle \
+    --bundle-sourcemap index.android.bundle.map
```

Comment:
```suggestion
  --rewrite index.android.bundle index.android.bundle.map
```
This is incorrect, as `bundle-sourcemap` expects a RAM bundle, and using it for normal source map bundles will show a warning: 
<img width="547" alt="Screen Shot 2021-03-25 at 1 49 23 AM" src="https://user-images.githubusercontent.com/30991498/112367313-5596c400-8d0c-11eb-8eb9-c0616c6ddfe4.png">
The original method of passing each file path was correct. We also should add the `--rewrite` flag for React Native sourcemaps.

Reply:
@jennmueng I haven't seen this warning in my experiments neither I'm using RAM bundle. Which version of CLI has this warning?

Reply:
I'm using  `@sentry/cli@1.63.1` and this is my command for bundle:

```
	NODE_OPTIONS=--max_old_space_size=4096 yarn react-native bundle \
		--entry-file $(BUNDLE_ENTRY_FILE) \
		--bundle-output $(BUNDLE_OUTPUT_FILE) \
		--sourcemap-output $(BUNDLE_SOURCEMAPS_OUTPUT_FILE) \
		--assets-dest $(BUNDLE_OUTPUT_FOLDER) \
		--platform $(K_PLATFORM) \
		--dev $(K_BUNDLE_DEV) \
		--reset-cache
```

Reply:
anyways. this seems to be working too, my current command is:

```
node_modules/@sentry/cli/bin/sentry-cli \
    releases \
    --org <org> \
    --project <project> \
    files <my release> \
    upload-sourcemaps \
    --dist <my dist> \
    --strip-prefix $(pwd) \
    --rewrite app.bundle/main.jsbundle \
     app.bundle/code.bundle.map
```


I guess main confusion was how to specify bundle and sourcemaps 🤔 and in which order

thanks!